portal-impl/src/com/liferay/portal/dao/orm/common/SQLTransformer.java

```diff
@@ -60,19 +61,21 @@ private SQLTransformer() {
 	}
 
 	private void _reloadSQLTransformer() {
-		if (_transformedSqls == null) {
-			_transformedSqls = new ConcurrentHashMap<>();
+		if (_transformedSqlsPortalCache == null) {
```

Comment:
@marianoalvarosaiz Please make `_transformedSqlsPortalCache` a `final` field, and remove the `if` part here, only keeping the `removeAll()` call, because you've already created the cache at the class creation.

Reply:
@dantewang I have delayed the creation to be consistent with `com.liferay.portal.dao.sql.transformer.SQLTransformer` where we are doing the same, see https://github.com/liferay-core-infra/liferay-portal/pull/4547